tests/acme_diags/derivations/test_acme.py

```diff
@@ -0,0 +1,80 @@
+from typing import TYPE_CHECKING
+from unittest import TestCase
+from unittest.mock import Mock
+
+import numpy as np
+
+from acme_diags.derivations.acme import adjust_prs_val_units, determine_cloud_level
+
+if TYPE_CHECKING:
+    from cdms2.axis import FileAxis
+
+
+class TestAdjustPrsValUnits(TestCase):
+    def setUp(self):
+        self.mock_file_axis: "FileAxis" = Mock()
+        self.mock_file_axis.getData.return_value = np.arange(0, 1002)
```

Comment:
This mocks the `cdms2.axis.FileAxis.getData()` method, which returns a `np.ndarray`